index.html

```diff
@@ -56,5 +56,190 @@ <h1 class="Intro">
         </li>
       </ul>
     </section>
+
+    <section class="work--section">
+      <div class="parent--container">
+        <div class="first--child">
+          <a href="#"><img src="images/2nd pic.svg" alt="" /></a>
+        </div>
+
+        <div class="second--child">
+          <h2>Tonic</h2>
+          <ul class="ul--list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="skill--list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="see--project">See project
+          </button>
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please follow the Figma design but in your case, your buttons and button effects are totally different so kindly follow the Figma design :+1: check below :point_down: the comparison between your design and Figma's design :slightly_smiling_face:


| Your Design | Figma Design|
|--------|--------|
| ![Screenshot from 2023-06-03 23-33-55](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/45226848/dd608ca9-b9a2-480b-9d46-20814dfb6c33) | ![Screenshot from 2023-06-03 23-34-04](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/45226848/684c10ac-8b59-40db-8024-2fe51a855136) |  

**Correct Figma design button effects**
![Screenshot from 2023-02-17 17-31-04](https://github.com/Ayokunnumi1/Portfolio-setup-and-mobile-first/assets/45226848/664d17cf-a2d0-42d5-b326-4312bfa948e7)